core/app/sorters/spree/base_sorter.rb

```diff
@@ -0,0 +1,35 @@
+module Spree
+  class BaseSorter
```

Comment:
is this valid to add spec for that sorter ?

Reply:
better to rely on request specs